ciao-controller/client.go

```diff
@@ -267,6 +267,31 @@ func (client *ssntpClient) RestartInstance(instanceID string, nodeID string) err
 	return err
 }
 
+func (client *ssntpClient) AssignFloatingIP(floatingIP payloads.FloatingIP) error {
+	floatingIPCmd := payloads.PublicIPCommand{
+		PublicIP:     floatingIP.AssignFloatingIP.Address,
+		PrivateIP:    floatingIP.AssignFloatingIP.InternalIP,
+		TenantUUID:   floatingIP.AssignFloatingIP.TenantUUID,
+		InstanceUUID: floatingIP.AssignFloatingIP.InstanceUUID,
```

Comment:
We should make providing the InstanceUUID optional. When we support k8s services, there will not be a instance behind the InternalIP. Hence allow for the case where the InstanceUUID is left un specified.